apis/concierge/authentication/v1alpha1/types_jwtauthenticator.go.tmpl

```diff
@@ -5,6 +5,19 @@ package v1alpha1
 
 import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
+type JWTAuthenticatorPhase string
```

Comment:
Copyright year is not updated in this file. Don't forget that when you change a .tmpl file you also need to run hack/update.sh.

Reply:
Even when just changing the copyright, thats interesting. 👍